ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/hardware_counters.schema.yml

```diff
@@ -0,0 +1,131 @@
+                features:
+                  - name: ip
+                    direction: out
+                    layer3: true
+                    units_packets: true
+              ```
+
+              to render:
+              ```eos
+              hardware counter feature ip out layer3 units packets
+              ```
+            * Use:
+              ```yaml
+              hardware_counters:
+                features:
+                  - name: route
+                    address_type: ipv4
+                    vrf: test
+                    prefix: 192.168.0.0/24
+              ```
+
+              to render:
+              ```eos
+              hardware counter feature route ipv4 vrf test 192.168.0.0/24
+              ```
+
+        # primary_key: name
+        # no primary key to allow multiple times the same name
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+              valid_values:
+                - acl
+                - decap-group
+                - directflow
+                - ecn
+                - flow-spec
+                - gre tunnel interface
+                - ip
+                - mpls interface
+                - mpls lfib
+                - mpls tunnel
+                - multicast
+                - nexthop
+                - pbr
+                - pdp
+                - policing interface
+                - qos
+                - qos dual-rate-policer
+                - route
+                - routed-port
+                - subinterface
+                - tapagg
+                - traffic-class
+                - traffic-policy
+                - vlan
+                - vlan-interface
+                - vni decap
+                - vni encap
+                - vtep decap
+                - vtep encap
+            direction:
+              type: str
+              valid_values:
+                - in
+                - out
+                - cpu
+              description: |
+                Most features support only 'in' and 'out'. Some like traffic-policy support 'cpu'.
+                Some features DO NOT have any direction.
+                This validation IS NOT made by the schemas.
+            address_type:
+              type: str
+              valid_values:
+                - ipv4
+                - ipv6
+                - mac
+              description: |
+                Supported only for the following features:
+                - acl: [ipv4, ipv6, mac] if direction is 'out'
+                - multicast: [ipv4, ipv6]
+                - route: [ipv4, ipv6]
+                This validation IS NOT made by the schemas.
+            layer3:
+              type: bool
+              description: |
+                Supported only for the 'ip' feature
+            vrf:
+              type: str
+              description: |
+                Supported only for the 'route' feature.
+                This validation IS NOT made by the schemas.
```

Comment:
```suggestion
```

Reply:
Using $ref to point to the `eos_cli_config_gen` schema.